modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationSpi.java

```diff
@@ -4382,20 +4384,11 @@ private UUID safeLocalNodeId() {
      * @return {@code True} if remote nodes support {@link HandshakeWaitMessage}.
      */
     private boolean isHandshakeWaitSupported() {
```

Comment:
what is the purpose for this change ?

Reply:
It's common logic for check feature support. I move it to the IgniteFeatures class.